modules/gentoo/2023.lua

```diff
@@ -12,6 +12,14 @@ if not arch or arch == "" then
 		arch = get_highest_supported_architecture()
 	end
 end
+if arch ~= "avx2" and arch ~= "avx512" then
```

Comment:
Don't we rather want to have 
if arch == "sse3" or arch == "avx" then 

For future compatibility ? (x86-64-v5 ?)

Reply:
I'd like to keep this mostly for symmetry with the `if` clause in StdEnv/2023. Better to mention arches that we do support than those we don't (there is no literal "sse3" or "avx" anywhere outside comments in 2023 files)